teacher.rb

```diff
@@ -0,0 +1,12 @@
+require relative 'person'
+
+class Teacher < Person
+  def initialize(age, specialization, name = 'Unknown')
```

Comment:
- [ ] It will be great to add `parent_permission: true` to the constructor. Remember you are using it when you call the `super` class below.